spec/controllers/media_controller_spec.rb

```diff
@@ -15,7 +22,7 @@
     scrubber = double('scrubber', save: true, url: 'http://www.google.com')
     expect(MediaScrubber).to receive(:new).and_return(scrubber)
 
-    post :create, use_route: :atomic_cms, file: double('file')
+    post :create, file: double('file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.